### PR TITLE
LPS-163756

### PR DESCRIPTION
**Pull request by murilostodolni**
https://issues.liferay.com/browse/LPS-163756

### CODE REVIEW

**Comment by murilostodolni** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/notification/term/contributor/ObjectDefinitionNotificationTermContributor.java [hunk 1]
```diff
@@ -85,7 +86,11 @@ public String getTermValue(Locale locale, Object object, String termName)
 			return termName;
 		}
 
-		return String.valueOf(termValues.get(objectField.getName()));
+		if (Validator.isNotNull(termValues.get(objectField.getName()))) {
```
Comment (rodrigopaulino) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/notification/term/contributor/ObjectDefinitionNotificationTermContributor.java:89]: Can you put `termValues.get(objectField.getName())` in a variable?

**Review by rodrigopaulino — CHANGES_REQUESTED** [review]

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:relevant

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:forward

**Review by rodrigopaulino — APPROVED** [review]
That's great, @murilostodolni. Thanks!

**Comment by murilostodolni** [comment]
ci:reevaluate:1297401_4781

**Comment by murilostodolni** [comment]
ci:forward

**Comment by rodrigopaulino** [comment]
Hey, @murilostodolni. This can be manually forward to Brian. 🙂

**Comment by rodrigopaulino** [comment]
Manually forwarded: https://github.com/brianchandotcom/liferay-portal/pull/123819

**Comment by murilostodolni** [comment]
Hi @rodrigopaulino thank you.

**Pull request closed without merging**